README.md

```diff
@@ -169,10 +170,63 @@ ATTR{suspend_mode}=="[oem|keep|off]"
 ```
 
 As en example, to set `suspend_mode`to `off`:
-`ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
+`ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
 `
 
 to create the rule in one line:
 ```shell
-$ echo 'ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
+$ echo 'ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
 ```
+
+### Bypass Charge Control
+
+Bypass Charge can be enabled, disabled, start and stop battery level can be set via sysfs.
+
+n most systems this will be mounted at `/sys/class/power_supply/ayaneo:bypass_charge/` and provides the following files:
+
+#### charge_type
+
+Read/write.
+
+Enables or disables the bypass charge. Accepts a value of 6 and 0 or "Custom" and "Unknown".
+
+|Value|Description|
+|-|-|
+|6 or "Custom"|Enables the bypass charge. Battery charging will depend on the start and stop values.|
```

Comment:
I'd personally prefer he names "bypass" and "normal". As an alternative to this and the above suggestions, we could have one single attribute called 'charge_control_enabled` that accepts 0/1 and is converted to 0/6 on the driver.

Reply:
The kernel enum/naming has "normal" and "bypass", but "cycle" would be a new name.
What do you mean with cycle? What intended behaviour should happen?

Reply:
Based on my understanding there are three modes of operation, correct?
Normal would be charge until 100% and charge below 100%
Bypass would be charge until X% and charge below X%
Cycle (or whatever we call it) would be charge until X% and charge below Y%.

Reply:
Looked into this more. Using the existing conventions it would be normal, bypass(upper and lower) and limit(upper only)

Reply:
I had read this: [sysfs-class-power](https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-power)
There they only name Standard, Custom and Bypass. Where should I look closer?
Do we not want to enable the bypass charge instantly, I mean we can also with the above method, but you have to set a limiting value.

Reply:
IDK what I read, this is correct. I guess it would be two modes, standard and bypass.